data-serving/scripts/prune-uploads/hooks/country_export.py

```diff
@@ -1,15 +1,16 @@
 # Exports country specific data by submitting
 # corresponding exporter_* job definitions
 
+from functools import cache
```

Comment:
I don't see what motivated this line or line 8 additions, was that your IDE?

Reply:
In these scripts imports were not separated by group (standard library, third party, local library) per https://www.python.org/dev/peps/pep-0008/#imports so I changed those, and in the process reorganized groups in alphabetical order (PEP8 is silent on this, but it makes it easier for me to read).